Removing Gradle plugin module (#252)
elastic/apm-agent-android: docs/faq.asciidoc

[[faq-how-does-it-work]]
=== How does the agent work?

> The agent is built on top of https://github.com/open-telemetry/opentelemetry-android[OpenTelemetry Android] and works as an extension that makes it easier to configure and set it all up to work with the Elastic tools with minimum overhead.

Reviewer: ```suggestion
The agent is built on top of https://github.com/open-telemetry/opentelemetry-android[OpenTelemetry Android]. It works as an extension, making it easier to set up and configure Elastic APM with minimum overhead.
```

> All the data created using the agent, either automatically or manually, is sent to the APM Server which converts it to a format suitable for Elasticsearch, and sends them to an Elasticsearch cluster.

Reviewer: ```suggestion
All data created by the agent, either automatically or manually, is sent to the APM Server, which converts it to a format suitable for Elasticsearch and sends it to an Elasticsearch cluster.
```

You can then use the APM app in Kibana to gain insight into latency issues and error culprits within your application and more.

[float]
[[faq-bytecode-instrumentation]]
=== Is the agent doing bytecode instrumentation?

> No, although if you'd like to get automatic instrumentations, you can add https://github.com/open-telemetry/opentelemetry-android/tree/main/auto-instrumentation[extensions] to do so that make use of bytecode instrumentation.

Reviewer: ```suggestion
No, although if you'd like to get automatic instrumentations, you can add https://github.com/open-telemetry/opentelemetry-android/tree/main/auto-instrumentation[extensions] that make use of bytecode instrumentation.
```

[float]
[[faq-metrics-aggregation]]
=== Metrics aggregation strategy

The metrics aggregation strategy used by the agent is https://github.com/open-telemetry/opentelemetry-java/blob/976edfde504193f84d19936b97e2eb8d8cf060e2/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/AggregationTemporality.java#L15[DELTA].

[float]
[[faq-unsupported-technologies]]
=== What if the agent doesn't support the technologies I'm using?

> The agent is built on top of https://opentelemetry.io/docs/instrumentation/java/getting-started/[OpenTelemetry] You can use its public API to create custom spans and transactions.
> You should also take a look at the https://github.com/open-telemetry/opentelemetry-android/tree/main/auto-instrumentation[automatic instrumentation extensions] currently available where the tool you need to instrument might be supported already.

Reviewer: ```suggestion
The agent is built on top of https://opentelemetry.io/docs/instrumentation/java/getting-started/[OpenTelemetry]. You can use its public API to create custom spans and transactions.
You should also take a look at the https://github.com/open-telemetry/opentelemetry-android/tree/main/auto-instrumentation[automatic instrumentation extensions] currently available where the tool you need to instrument might be supported already.
```
